riff-raff/public/docs/riffraff/administration/auth.md

```diff
@@ -8,6 +8,19 @@ Authentication
 Authentication is handled by Google. This obtains the users full name and e-mail address and both of these are used to 
 provide an audit trail of deployments and other actions in the app.
 
+Google auth is implemented in two layers
+- Load Balancer layer: this component is defined in the main riff raff cloudformation template and uses [authenticate-oidc](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#configure-user-authentication). The template relies on a secret stored in SSM.
+- Application layer: this layer uses The Guardian's [Play Google Auth Module](https://github.com/guardian/play-googleauth) and is configured in the main [application.conf](https://github.com/guardian/riff-raff/blob/afb7e602e11acd7a07aae433c74be22976d8a7cd/riff-raff/conf/application.conf#L40-L41).
+
+To rotate the secrets, follow these steps:
+1. Create a new Client Secret in Riff Raff's Google Cloud project
+2. Patch the PROD and CODE application.conf to use the new secret
+3. Update the secret in AWS Secrets Manager used by the cloudformation template.
```

Comment:
I was expecting that we'd need to somehow force the CFN stack to update too in order to [re-resolve the secret value](https://github.com/guardian/deploy-tools-platform/blob/1edf6e913150a136efa6ee14c2b9f100f00fe923/cloudformation/riffraff/riffraff.template.yaml#L592).

@akash1810 - was that necessary earlier this week?

Reply:
Ah yes, I can see that [a manual step was required](https://docs.google.com/document/d/17rPcdh2KUffO9s2rpkbOSv1DzgOFuhkkEOQI3W4pCGQ/edit?tab=t.0#bookmark=id.id24n9xuqxfx) so we should try to capture this.

Reply:
https://github.com/guardian/deploy-tools-platform/pull/872 should help with this.